install/docker-extension/README.md

```diff
@@ -60,6 +60,26 @@ The Docker Extension for Meshery deploys Meshery to your local Docker host as a
 <p align="center"><a href="https://raw.githubusercontent.com/meshery/meshery/master/install/docker-extension/docs/img/docker-extension-for-meshery-architecture.png"><img src="https://raw.githubusercontent.com/meshery/meshery/master/install/docker-extension/docs/img/docker-extension-for-meshery-architecture.png" width="90%" align="center" /></a></p>
 Learn more about <a href="https://docs.meshery.io/architecture">Meshery's architecture</a>.
 
+## Docker Extension for Meshery
+
+From `/install/docker-extension`, familiarize with available `make` targets by executing:
+
+```
+make
+```
+Review the available targets and their purpose. In general, follow this sequence when building and testing changes:
+
+```
+make extension
+```
+
+Once build is complete:
+
+```
+docker extension install meshery/docker-extension-meshery:edge-latest
+```
+
```

Comment:
@leecalcote  To update the extension instead of having to remove and reinstall the extension again. You can use `docker extension update meshery/docker-extension-meshery:edge-latest` instead. I think this would be a welcome addition in the doc as well.

Reply:
Thanks, @anirudhjain75